crates/bevy_window/src/lib.rs

```diff
@@ -73,3 +73,6 @@ impl Plugin for WindowPlugin {
         }
     }
 }
+
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemLabel)]
+pub struct ModifiesWindows;
```

Comment:
Can systems have multiple labels, if not then that approach doesn’t really hold because a system could modify multiple things And the windows.

Reply:
Yes, multiple levels are permitted.

Reply:
Ok. Then this is not a problem.